src/backend/utils.ts

```diff
@@ -54,46 +55,10 @@ const { showMessageBox } = dialog
  * Checks if target is newer than base.
  */
 function semverGt(target: string, base: string) {
-  if (!target || !base) {
+  if (!semver.valid(target) || !semver.valid(base)) {
     return false
   }
-  target = target.replace('v', '')
-
-  // beta to beta
-  if (base.includes('-beta') && target.includes('-beta')) {
-    const bSplit = base.split('-beta.')
-    const tSplit = target.split('-beta.')
-
-    // same major beta?
-    if (bSplit[0] === tSplit[0]) {
-      base = bSplit[1]
-      target = tSplit[1]
-      return target > base
-    } else {
-      base = bSplit[0]
-      target = tSplit[0]
-    }
-  }
-
-  // beta to stable
-  if (base.includes('-beta')) {
-    base = base.split('-beta.')[0]
-  }
-
-  // stable to beta
-  if (target.includes('-beta')) {
-    target = target.split('-beta.')[0]
-  }
-
-  const [bmajor, bminor, bpatch] = base.split('.').map(Number)
-  const [tmajor, tminor, tpatch] = target.split('.').map(Number)
-
-  let isGE = false
-  // A pretty nice piece of logic if you ask me. :P
-  isGE ||= tmajor > bmajor
-  isGE ||= tmajor === bmajor && tminor > bminor
-  isGE ||= tmajor === bmajor && tminor === bminor && tpatch > bpatch
-  return isGE
+  return semver.gt(target, base)
```

Comment:
we need to test if the package covers all use cases. I am worried about when we release a stable from a previous beta.
like:
- 2.2.0-beta.1, then
- 2.2.0

Reply:
There are tests for this function, testing exactly that. They're all passing